pygmt/tests/test_grd2xyz.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for grd2xyz.
+"""
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from pygmt import grd2xyz
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(resolution="01d", region=[-1, 1, -1, 1])
+
+
+def test_grd2xyz(grid):
+    """
+    Make sure grd2xyz works as expected.
+    """
+    xyz_data = grd2xyz(grid=grid, output_type="numpy")
+    assert xyz_data.shape == (4, 3)
+
+
+def test_grd2xyz_format(grid):
+    """
+    Test that correct formats are returned.
+    """
+    xyz_default = grd2xyz(grid=grid)
+    assert isinstance(xyz_default, pd.DataFrame)
+    print(type(list(xyz_default.columns)))
+    assert list(xyz_default.columns) == ["x", "y", "z"]
```

Comment:
The earth_relief grid dimensions are `lat` and `lon`, with the 'z' variable being called `elevation`. Oh, and the print statement should be left out.

```suggestion
    assert list(xyz_default.columns) == ["lon", "lat", "elevation"]
```